src/control/run_go_tests.sh

```diff
@@ -2,6 +2,59 @@
 ## Run linters across control plane code and execute Go tests
 set -e
 
+function find_build_source()
+{
+	BASE=$PWD
+	while true
+	do
+		path=$(realpath "${BASE}")
+		if [ "${path}" == "/" ]; then
+			break
+		fi
+		test -e "${path}/.build_vars.sh" && echo "${path}/.build_vars.sh" && return
```

Comment:
Minor comment - Maybe the string "${path}/.build_vars.sh" could be a variable since it's used twice here? Would probably knock the line under 80 chars too

Reply:
Good idea. I'm not sure why but I referred to it as build_source.sh later on for some odd reason so I'll have to change that as well.

Reply:
Good suggestion. Fixed in latest patch.